file-browser/src/services/dropbox.js

```diff
@@ -35,3 +39,20 @@ export function* dropboxMainProc() {
     // fork(new_process, arguments1, arg2, arg3...);
     // }
 }
+
+
+const uuidV4 = require('uuid/v4');
+export function accountHelper(cliendId = testClientId, accessToken = testAccessToken, uuidGenerator = uuidV4) {
```

Comment:
this shouldn't been account helper. 

- [ ] we need: `accountKey` helper

uuidGenerator shouldn't be there either.